integration-tests/pom.xml

```diff
@@ -83,6 +83,41 @@
           <skip>true</skip>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-failsafe-plugin</artifactId>
+        <configuration>
+          <systemProperties>
+            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+            <!--
+                Make use of JUnit 5 tags instead
+                See https://github.com/cucumber/cucumber-jvm/tree/main/junit-platform-engine#tags
+            -->
+            <!--cucumber.filter.tags>${tags}</cucumber.filter.tags-->
+
+            <!-- https://github.com/cucumber/cucumber-jvm/tree/main/junit-platform-engine#parallel-execution -->
+            <cucumber.execution.parallel.enabled>${parallel}</cucumber.execution.parallel.enabled>
+            <cucumber.junit-platform.naming-strategy>long</cucumber.junit-platform.naming-strategy>
+            <cucumber.execution.parallel.config.fixed.parallelism>${parallel.concurrency.number}</cucumber.execution.parallel.config.fixed.parallelism>
+
+            <!-- Token authorization configuration-->
+            <bridge.client.id>kermit</bridge.client.id>
+            <bridge.client.secret>N5TW1EfuIcQsplRsLXJ1aE3DZZMPN5ZH</bridge.client.secret>
```

Comment:
Do we want to hard code this parameter?

Reply:
This value is taken from https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json to simplify usage and it is relatively complicate to share it just on one place.
So, I would say yes, in this case it is ok to hardcode it here.

Reply:
@sutaakar  shouldn't we share this information as comment in pom.xml? I mean where `N5TW1EfuIcQsplRsLXJ1aE3DZZMPN5ZH` is coming from?

Reply:
Yes, good idea.

Reply:
Created https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/674